azerothcore/azerothcore-wotlk
# fix(DB/ItemLoot): Membership Benefits item loot adjustment

## Changes Proposed:
-  fixes no loot sometimes
-  added ref tables
- added missing gem, Deep peridot

## Issues Addressed:

- Partially closes https://github.com/azerothcore/azerothcore-wotlk/issues/15537
- Closes https://github.com/chromiecraft/chromiecraft/issues/4666

## SOURCE:


_wowhead, i am sorry, nobody wants to sniff this_
all the youtube videos I could find

## Tests Performed:

- Tested


## How to Test the Changes:


1. Add the several bags to inventory
2. see its never empty

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
